docs/documentation/tips-tricks/page-redirects.rst

```diff
@@ -0,0 +1,34 @@
+Handle page redirects
```

Comment:
Could we please remove this page? I'm opening #882 to take a less hacky approach to this problem by maintaining a list of redirects in configuration in the project.